src/dcc/Packet.cxxtest

```diff
@@ -335,6 +335,45 @@ TEST_F(PacketTest, DccBasicAccyOn)
     EXPECT_THAT(get_packet(), ElementsAre(b1, b2, b1 ^ b2));
 }
 
+TEST(DefsTest, AccyAddress)
+{
+    for (unsigned user = 1; user <= 2048; ++user)
+    {
+        SCOPED_TRACE(user);
+        // Inversion should be OK.
+        EXPECT_EQ(user,
+            Defs::accy_address_binary_to_user(
+                Defs::accy_address_user_to_binary(user)));
+    }
+    for (unsigned binary = 0; binary < 2048; ++binary)
```

Comment:
Would it be clearer to use <= 2047 to be consistent with the previous for loop?

Reply:
done